helmfile.d/stacks/ingress-nginx.yaml

```diff
@@ -5,14 +5,24 @@ templates:
     labels:
       app: ingress-nginx
 
+  file-copier:
+    inherit: [ template: ingress-nginx ]
+    name: file-copier
+    namespace: kube-system
+    chart: charts/file-copier
+    version: 0.1.0
+    installed: {{ .Values | get "ingressNginx.controller.chroot" false }}
+    values:
+      - values/file-copier.yaml.gotmpl
+
```

Comment:
I was thinking that "file-copier" as a chart could be separated from ingress-nginx, if we want to copy more files onto hosts (e.g. public ssh keys as you mentioned) we could just put more daemonsets/configmaps in it.

Reply:
Yes, I'm thinking we'll move it to system once it is more generic.